lib/models/movie.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```

Comment:
In this particular case we _cannot_ use an arrow function ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)) because it does not bind `this`. Using an arrow function here would mean we wouldn't be able to access the properties through `this.get('title')` as `this` is not bound to the model itself.